packages/components/src/Slider/Slider.scss

```diff
@@ -0,0 +1,95 @@
+$tc-slider-thumb-shadow: 0 1px 2px 0 rgba(117, 132, 149, 0.5); //$slate-gray;
+$tc-slider-thumb-shadow-focused: 0 3px 4px 0.5px rgba(0, 0, 0, 0.25);
+$tc-slider-font-size: $font-size-base;
+$tc-slider-line-height: 24px;
+
+:global(.rc-slider-tooltip) {
+	padding: 0;
+
+	:global(.rc-slider-tooltip-arrow) {
+		display: none;
+	}
+
+	:global(.rc-slider-tooltip-inner) {
+		background: transparent;
+		color: $brand-primary;
+		font-size: $tc-slider-font-size;
+		box-shadow: 0 0 0px;
+	}
+}
+
+.tc-slider-container {
+	display: block;
+
+	:global(.rc-slider-disabled) {
+		background-color: transparent;
+		opacity: 0.4;
+	}
+
+	& label {
+		display: block;
+	}
+
+	.informations {
+		display: flex;
+		& svg {
+			height: $tc-slider-line-height;
+			width: $tc-slider-line-height;
+			fill: $brand-primary;
+		}
+
+		.informations-label {
+			margin-left: $padding-small;
+			line-height: $tc-slider-line-height;
+		}
+	}
+
+	.text-caption {
+		display: flex;
+		justify-content: space-between;
+		margin-top: $padding-small;
+		color: $gray;
+		font-size: $tc-slider-font-size;
+		font-weight: 400;
+	}
+
+	.icons {
```

Comment:
you can merge this one with `.text-caption`

```
	.text-caption,
	.icons {
		display: flex;
		justify-content: space-between;
		margin-top: $padding-small;
		color: $gray;
		font-size: $tc-slider-font-size;
		font-weight: 400;

		:global(svg) {
			height: $tc-slider-line-height;
			width: $tc-slider-line-height;
			fill: $gray;
		}
		.selected {
			fill: $brand-primary;
		}
	}
```

Even better, define only 1 classname for both : `captions`

Reply:
Still KO